understanding/22/focus-appearance.html

```diff
@@ -13,7 +13,16 @@ <h2>Status</h2>
         <p>This understanding document is part of the <a href="https://w3c.github.io/wcag/guidelines/22/"><strong>draft</strong> WCAG 2.2 content</a>. It may change or be removed before the final WCAG 2.2 is published.</p>
     </section>
 
+    <section id="brief">
+        <h2>In brief</h2>
+        <dl>
+            <dt>Goal</dt><dd>Make it easier to spot the keyboard focus.</dd>
+            <dt>What to do</dt><dd>Use a focus indicator of sufficient size and contrast.</dd>
+            <dt>Why it's important</dt><dd>Many people can't see small contrast differences, including older people.</dd>
```

Comment:
```suggestion
            <dt>Why it's important</dt><dd>Many people can't see small or low contrast differences, including older people.</dd>
```

Reply:
Hmm, I don't think they meant "small contrast" here to mean small size... How about
```suggestion
            <dt>Why it's important</dt><dd>Many people can't see small changes in contrast, including older people.</dd>
```

Reply:
ah, but then...the "sufficient size" part of the "what to do" isn't really touched on

Reply:
why not? A smaller change is more subtle. The Why is supposed to be illustrative, not a 1:1 response to each point in the wording of the goal or normative text.

Reply:
I think "subtle changes in visual appearance" is not specific enough and also uses bigger words and more abstract concept

Reply:
changing to "small changes", which I think addresses both concerns.